### DISCUSSION TITLE
Please check if gene name/symbol and description are correct for the Ensembl gene IDs on the portal

### DISCUSSION THREAD

**Discussion by mano-at-sdsc** [nih-cfde/published-documentation#420] [Bug]
Dear CC Team:

It appears that the gene name/description displayed for gene search do not correspond to the Ensembl ID listed. Please check it out.

Example 1:

https://app.nih-cfde.org/chaise/record/#1/CFDE:gene/nid=10613

says HK1 for ENSG00000152894, which is incorrect.

Just the gene name/symbol and description seem wrong, everything else seems to be fine.

Example 2:

https://app.nih-cfde.org/chaise/record/#1/CFDE:gene/nid=8136

ID: ENSG00000135443

Name: ALDOB, which is incorrect
Description: aldolase, fructose-bisphosphate B [Source:HGNC Symbol;Acc:HGNC:417], incorrect

Thanks.

Mano

**Comment by mgiglio99** [comment]
Hi Mano,
Thanks for alerting us to the problem - we're checking into it. 
Michelle
